### ISSUE TITLE
Remove LinuxRelease() in provisionLinuxForLisa.sh

### ISSUE THREAD

**Issue by juhlee-microsoft** [LIS/LISAv2#201]
Replace with GetDistro from utils.sh

**Comment by chvalean** [comment]
@juhlee-microsoft @iamshital I think we just have to remove the entire script provisionLinuxForLisa.sh
It is no longer required as tests will be imported in lisav2, so there is no VM preparation required for lisa.

Let's just do that and close this item.

**Comment by lizzha** [comment]
this script is used in function ProvisionVMsForLisa, it enables root user and creates/upload same SSH key to client and server, which is required by network performance cases. It's still needed.
It's better to change a function/script name.

**Comment by juhlee-microsoft** [comment]
According to @lizzha, we still need to change. Please keep this item in backlog.

**Comment by chvalean** [comment]
I verified that function and although the provisionLinuxForLisa.sh script is copied to the VM, I don't see it getting executed from the function code.
@iamshital  can you please verify the code and maybe do some cleanup if provisionLinuxForLisa.sh is not actually used?

**Comment by lizzha** [comment]
@chvalean you're right, the script is not executed. Only enableRoot.sh and enablePasswordLessRoot.sh are executed. So it can be removed.
Still we'd better change a function name of ProvisionVMsForLisa.
